src/force/LennardJonesMolecule.h

```diff
@@ -0,0 +1,67 @@
+#include "Force.h"
+
+/** \brief
+ *  The Lennard-Jones potential is a pairwise potential that describes the
+ *  interaction between two particles. It is used to model the interaction
+ *  between atoms or molecules.
+ *  The Lennard-Jones potential is a pairwise potential, so the force is only
+ *  calculated between two particles.
+ */
+class LennardJonesMolecule : public Force {
```

Comment:
I would still pack "Force" into the name of `LennardJonesMolecule`. Otherwise, the class sounds misleadingly like a molecule/particle class.

Reply:
I get your point and from an outside standpoint I could see the issue arise, but first it is located in a folder named force/ and second I don't know a name that would be descriptive, contain force and stays reasonably long.
I am open for suggestions here though.

But if you still wish for change I then would change Planet .cpp/.h as well as the same issue arises here.

Reply:
What about `LennardJonesForce`? It should be obvious to a user that the lennard jones force should be applied in the context of molecules.